airflow/models/dagrun.py

```diff
@@ -349,9 +345,14 @@ def update_state(self, session=None):
 
         return ready_tis
 
-    def _get_ready_tis(self, scheduleable_tasks, finished_tasks, session):
+    def _get_ready_tis(
+        self,
+        scheduleable_tasks: List[TI],
+        finished_tasks: List[TI],
+        session,
```

Comment:
from sqlalchemy.orm.session import Session
```suggestion
        session: Session,
```